torchao/prototype/safetensors/safetensors_support.py

```diff
@@ -136,8 +126,4 @@ def save_tensor_state_dict(
         combined_tensors_dict.update(prefixed_tensors_dict)
 
     combined_metadata["tensor_names"] = json.dumps(list(tensor_dict.keys()))
-
-    save_file(combined_tensors_dict, file_path, metadata=combined_metadata)
-    logger.info(
-        f"Saved {len(tensor_dict)} tensor subclasses to {file_path} with metadata"
-    )
+    return combined_tensors_dict, combined_metadata
```

Comment:
so `combined_tensors_dict` corresponds to `tensor_data` in the other function?

I think we should add some more docs for the returned values here, ideally with some examples, also for the input args for `load_tensor_state_dict`

also will need to rename `load_tensor_state_dict` and `save_tensor_state_dict` to reflect the functionality changes